webextensions/api/find.json

```diff
@@ -22,6 +22,48 @@
               },
               "safari_ios": "mirror"
             }
+          },
+          "options": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "57"
+                },
+                "firefox_android": {
+                  "version_added": false
+                },
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            },
+            "matchDiacritics": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": false
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": "111"
+                  },
+                  "firefox_android": {
+                    "version_added": false
+                  },
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": false
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            }
```

Comment:
```suggestion
          },
          "options": {
            "__compat": {
              "support": {
                "chrome": {
                  "version_added": false
                },
                "edge": "mirror",
                "firefox": {
                  "version_added": "57"
                },
                "firefox_android": {
                  "version_added": false
                },
                "opera": "mirror",
                "safari": {
                  "version_added": false
                },
                "safari_ios": "mirror"
              }
            },
            "tabId": {
              "__compat": {
                "support": {
                  "chrome": {
                    "version_added": false
                  },
                  "edge": "mirror",
                  "firefox": {
                    "version_added": "57"
                  },
                  "firefox_android": {
                    "version_added": false
                  },
                  "opera": "mirror",
                  "safari": {
                    "version_added": false
                  },
                  "safari_ios": "mirror"
                }
              }
            },
            "caseSensitive": {
              "__compat": {
                "support": {
                  "chrome": {
                    "version_added": false
                  },
                  "edge": "mirror",
                  "firefox": {
                    "version_added": "57"
                  },
                  "firefox_android": {
                    "version_added": false
                  },
                  "opera": "mirror",
                  "safari": {
                    "version_added": false
                  },
                  "safari_ios": "mirror"
                }
              }
            },
            "entireWord": {
              "__compat": {
                "support": {
                  "chrome": {
                    "version_added": false
                  },
                  "edge": "mirror",
                  "firefox": {
                    "version_added": "57"
                  },
                  "firefox_android": {
                    "version_added": false
                  },
                  "opera": "mirror",
                  "safari": {
                    "version_added": false
                  },
                  "safari_ios": "mirror"
                }
              }
            },
            "includeRangeData": {
              "__compat": {
                "support": {
                  "chrome": {
                    "version_added": false
                  },
                  "edge": "mirror",
                  "firefox": {
                    "version_added": "57"
                  },
                  "firefox_android": {
                    "version_added": false
                  },
                  "opera": "mirror",
                  "safari": {
                    "version_added": false
                  },
                  "safari_ios": "mirror"
                }
              }
            },
            "includeRectData": {
              "__compat": {
                "support": {
                  "chrome": {
                    "version_added": false
                  },
                  "edge": "mirror",
                  "firefox": {
                    "version_added": "57"
                  },
                  "firefox_android": {
                    "version_added": false
                  },
                  "opera": "mirror",
                  "safari": {
                    "version_added": false
                  },
                  "safari_ios": "mirror"
                }
              }
            },
            "matchDiacritics": {
              "__compat": {
                "support": {
                  "chrome": {
                    "version_added": false
                  },
                  "edge": "mirror",
                  "firefox": {
                    "version_added": "111"
                  },
                  "firefox_android": {
                    "version_added": false
                  },
                  "opera": "mirror",
                  "safari": {
                    "version_added": false
                  },
                  "safari_ios": "mirror"
                }
              }
            }
```

Never used Github's review feature before so I'm not sure how this will look.

Reply:
@gregorypappas my understanding is that it's normal practice only to document exceptions in sub features, however, as you have added these I am adding them in.